dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/publish/servers/Telemetry10/server.xml

```diff
@@ -3,11 +3,8 @@
 	<include location="../fatTestPorts.xml" />
 
 	<featureManager>
-		<feature>cdi-3.0</feature>
-		<feature>mpConfig-3.0</feature>
-		<feature>servlet-5.0</feature>
-        <feature>mpRestClient-3.0</feature>
-		<feature>mpTelemetry-1.0</feature>
+		<feature>servlet-6.0</feature>
+                <feature>mpTelemetry-1.0</feature>
```

Comment:
Is `mpRestClient=3.0` not needed for this FAT? It's not included as a dependency of `mpTelemetry-1.0`.

Reply:
The test uses restfulWS-3.1, which is pulled in automatically by mpTelemetry-1.0. It doesn't use mpRestClient-3.0.